### ISSUE TITLE
Provide swagger-diff as standalone tool

### ISSUE THREAD

**Issue by cezarypiatek** [Sayi/swagger-diff#6]
Hi
Could you publish this amazing library as standalone tool to make it much more easier to use? This tool should accept parameters such as OldSpecification, NewSpecification, RenderingMode, OutputFile.

**Comment by Sayi** [comment]
Sorry I didn’t have time recently.

**Comment by Sayi** [comment]
I has released `v1.2.0`, and add the feature of Command Line Interface.
```shell
$ java -jar swagger-diff.jar --help
Usage: java -jar swagger-diff.jar [options]
  Options:
  * -old
      old api-doc location:Json file path or Http url
  * -new
      new api-doc location:Json file path or Http url
    -v
      swagger version:1.0 or 2.0
      Default: 2.0
    -output-mode
      render mode: markdown or html
      Default: markdown
    --help

    --version
      swagger-diff tool version
```
[View Document](http://deepoove.com/swagger-diff/)
